src/gluonts/representation/mean_scaling.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from .representation import Representation
+
+import mxnet as mx
+
+
+# Standard library imports
+from typing import Tuple, Optional, List
+
+# First-party imports
+from gluonts.core.component import validated
+from gluonts.model.common import Tensor
+
+
+class MeanScaling(Representation):
+    """
+    A class representing a mean scaling approach.
+    Inputs are simply rescaled based on their mean.
+
+    Parameters
+    ----------
+    clip_max
+        The maximum value to which re-scaled values will be clipped to. By default, max values will not be clipped.
+        (default: -1)
+    minimum_scale
+        The minimum value to which re-scaled values will be clipped to.
+        (default: 1e-10)
+    """
+
+    @validated()
+    def __init__(
+        self, clip_max: int = -1, scale_min: float = 1e-10, *args, **kwargs
```

Comment:
Shouldn't `clip_max` be a `float`? If it can be left unspecified it's better to use `Optional[float]` and default it to `None`

Reply:
Fixed!